src/Omnipay/Common/Message/AbstractResponse.php

```diff
@@ -148,6 +148,16 @@ public function getTransactionReference()
     }
 
     /**
+     * Get the transaction ID as generated by the merchant website.
+     *
+     * @return string
+     */
+    public function getTransactionId()
```

Comment:
I think it is largely similar to transactionReference & having it treated the same makes sense. ie. in this cut it is sitting just below the equivalent function for getTransactionReference.

It won't always be provided, like transactionReference but it is easily & commonly confused with transactionReference (on clarifying the meaning through github issues I also had to go back & patch processors that has mixed up these 2 fields) - so putting it alongside transactionReference makes the relationship clearer to my mind.